src/index.ts

```diff
@@ -0,0 +1,33 @@
+import express, { Request, Response } from 'express';
+import os from 'os';
+
+const app = express();
+const port = 3000;
+app.use(express.json());
+
+app.set('trust proxy', true);
```

Comment:
What is this?

Reply:
tried to experiment if I want to see real ip if I wasn't testing locally and the ip was sent through the X-Forwarded-For header.  since the ip when testing locally shows as  "::1",